{u'description': [u'Gentoo cross-compilation overlay for embedded devices'],
 u'feeds': [u'https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 u'homepage': u'https://cgit.gentoo.org/proj/embedded-cross.git/',
 u'irc': None,
 u'name': u'embedded-cross',
 u'official': True,
 u'owner_email': u'armin76@gentoo.org',
 u'owner_name': u'Ra\xfal Porcel',
 u'priority': 50,
 u'quality': u'experimental',
 u'sources': [(u'git://anongit.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'https://anongit.gentoo.org/git/proj/embedded-cross.git',
               u'Git',
               u''),
              (u'git+ssh://git@git.gentoo.org/proj/embedded-cross.git',
               u'Git',
               u'')],
 u'status': u'official',
 u'supported': True}
$ pmaint sync /home/mgorny/repos/embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named 'clickbeetle_tools', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named 'armagetron', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/bicatali', named 'bicatali', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/docd', named 'docd', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/drizzt-overlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/drizzt-overlay', named '<unlabeled repository /home/mgorny/repos/drizzt-overlay>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/arcon' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/arcon', named '<unlabeled repository /home/mgorny/repos/arcon>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/dauleet' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/dauleet', named '<unlabeled repository /home/mgorny/repos/dauleet>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/armagetron', named 'armagetron', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/Menelkir', named 'Menelkir', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/clickbeetle_tools', named 'clickbeetle_tools', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/docd', named 'docd', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/alice', named 'alice', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at location '/home/mgorny/repos/drizzt-overlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/drizzt-overlay', named '<unlabeled repository /home/mgorny/repos/drizzt-overlay>', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
WARNING:pkgcore:repository at '/home/mgorny/repos/bicatali', named 'bicatali', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Already up-to-date.
*** syncing '/home/mgorny/repos/embedded-cross'...
*** synced '/home/mgorny/repos/embedded-cross'
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has an unsupported profile format(s): 'profile-set'

!!! ERROR: app-misc/mdbus-9999 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '165' 'depend'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'

!!! ERROR: dev-libs/libnl-2.0_pre9999 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '208' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: dev-python/pyrtc-20090418 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '218' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: sci-geosciences/fso-gpsd-0.8 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'

!!! ERROR: sys-devel/gcc-4.4.3 failed.

!!! ERROR: dev-libs/libgee-0.5.0.1 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'

!!! ERROR: media-plugins/gst-plugins-ivorbis-0.10.9 failed.
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'
 * Use PYTHON_DEPEND variable instead of NEED_PYTHON variable.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: dev-python/gst-python-0.10.16 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: dev-python/pyrtc-9999 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-devel/gcc-4.4.2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '189' 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                  ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '261' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!                  ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '215' 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * 
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
 * "/home/mgorny/repos/embedded-cross/dev-python/python-e_dbus/python-e_dbus-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '166' 'depend'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'

!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '204' 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                          ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libgee/libgee-0.5.0.1.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-20090418.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '169' 'depend'

!!! ERROR: dev-python/pygobject-2.18.0 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!                          ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!                 ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!          pyrtc-20090418.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/media-plugins/gst-plugins-ivorbis/gst-plugins-ivorbis-0.10.9.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '195' 'depend'
!!!          libgee-0.5.0.1.ebuild, line   5:  Called inherit 'gnome2'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pyrtc/pyrtc-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.3.ebuild'
!!!   gst-plugins-ivorbis-0.10.9.ebuild, line   5:  Called inherit 'gst-plugins-bad'
!!!              pyrtc-9999.ebuild, line   5:  Called inherit 'distutils' 'git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '228' 'depend'
!!!               gcc-4.4.3.ebuild, line  24:  Called inherit 'toolchain'
!!!                 ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'distutils'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 251' for eclass distutils

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
received unknown com: export CATEGORY='dev-python' PR=r0 PVR=20090729 PV=20090729 EBUILD='/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090729.ebuild' P='python-elementary-20090729' PF='python-elementary-20090729' PN='python-elementary'path
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/gst-python/gst-python-0.10.16.ebuild'
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins-bad'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: dev-python/python-ecore-20090729 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!      gst-python-0.10.16.ebuild, line   7:  Called inherit 'autotools' 'eutils' 'multilib' 'python'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-libs/libnl/libnl-2.0_pre9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-devel/gcc/gcc-4.4.2.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gnome2.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'python'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!                  gnome2.eclass, line  19:  Called die
!!! The die message:
!!!  EAPI= is not supported

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!       libnl-2.0_pre9999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/python.eclass'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                  python.eclass, line 277:  Called die
!!! The die message:
!!!  NEED_PYTHON variable is banned

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '226' 'depend'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-e_dbus/python-e_dbus-20091202.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sci-geosciences/fso-gpsd/fso-gpsd-0.8.ebuild'
!!!                 ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '248' 'depend'

!!! ERROR: dev-python/python-ecore-20090614 failed.
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins-bad.eclass'
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!              gst-plugins-bad.eclass, line  19:  Called inherit 'eutils' 'gst-plugins10'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!               gcc-4.4.2.ebuild, line  24:  Called inherit 'toolchain'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sci-geosciences/fso-gpsd/fso-gpsd-0.8.ebuild'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gst-plugins10'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!                   ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '203' 'depend'
!!!                 ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/gst-plugins10.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!                gst-plugins10.eclass, line  37:  Called die
!!! The die message:
!!!  EAPI="0" is not supported anymore

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sci-geosciences/fso-gpsd/fso-gpsd-0.8.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'toolchain'
!!!            fso-gpsd-0.8.ebuild, line   5:  Called inherit 'autotools' 'eutils'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                         ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!                         ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090729.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'autotools'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '226' 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 221' for eclass autotools

received unknown com: export CATEGORY='dev-python' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/dev-python/python-edje/python-edje-20090614.ebuild' P='python-edje-20090614' PF='python-edje-20090614' PN='python-edje'path
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!   python-ecore-20090729.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/toolchain.eclass'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!               toolchain.eclass, line  29:  Called die
!!! The die message:
!!!  Need to upgrade to at least EAPI=2

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090614.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090614.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 221' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='dev-python' PR=r0 PVR=20090614 PV=20090614 EBUILD='/home/mgorny/repos/embedded-cross/dev-python/python-evas/python-evas-20090614.ebuild' P='python-evas-20090614' PF='python-evas-20090614' PN='python-evas'path
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/pygobject/pygobject-2.18.0.ebuild'
!!!        pygobject-2.18.0.ebuild, line   5:  Called inherit 'alternatives' 'autotools' 'gnome2' 'python' 'virtualx'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'alternatives'
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 236' for eclass alternatives

received unknown com: export CATEGORY='dev-python' PR=r0 PVR=20090729 PV=20090729 EBUILD='/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20090729.ebuild' P='python-emotion-20090729' PF='python-emotion-20090729' PN='python-emotion'path
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-efl_utils/python-efl_utils-20090729.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 

!!! ERROR: dev-python/python-emotion-20091202 failed.
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                  ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-ecore/python-ecore-20090614.ebuild'
!!!             isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
 * 
 * "/home/mgorny/repos/embedded-cross/dev-python/python-elementary/python-elementary-20090614.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                   x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   python-ecore-20090614.ebuild, line   7:  Called inherit 'enlightenment_svn-python'
!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '236' 'depend'

!!! ERROR: dev-python/python-emotion-20090614 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/apm2-2.0.0 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 194' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='x11-libs' PR=r0 PVR='0.9.9.063' PV='0.9.9.063' EBUILD='/home/mgorny/repos/embedded-cross/x11-libs/ecore/ecore-0.9.9.063.ebuild' P='ecore-0.9.9.063' PF='ecore-0.9.9.063' PN=ecorepath
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-libs/libnl-2.0_pre9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: dev-python/pyrtc-20090418
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sci-geosciences/fso-gpsd-0.8
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pyrtc-9999
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/pygobject-2.18.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-ecore-20090729
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-elementary-20090729
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-libs/libgee-0.5.0.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/gst-python-0.10.16
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-ecore-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.3
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: media-plugins/gst-plugins-ivorbis-0.10.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-edje-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: dev-lang/vala-0.8.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-devel/gcc-4.4.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-evas-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-emotion-20090729
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-emotion-20091202
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-python/python-emotion-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/apm2-2.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/ecore-0.9.9.063
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsogsmd-0Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
 * 
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
 * "/home/mgorny/repos/embedded-cross/dev-python/python-epsilon/python-epsilon-20091202.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20091202.ebuild'
!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '236' 'depend'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20091202.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20091202.ebuild'
!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!   python-emotion-20091202.ebuild, line   5:  Called inherit 'enlightenment_svn-python'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '183' 'depend'
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20090614.ebuild'

!!! ERROR: x11-base/xglamo-20081205-r2 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: sys-mobilephone/fsomusicd-0.0.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20090614.ebuild'
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 276' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=9999 PV=9999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-9999.ebuild' P='freesmartphone-framework-9999' PF='freesmartphone-framework-9999' PN='freesmartphone-framework'path
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '201' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/dev-python/python-emotion/python-emotion-20090614.ebuild'

!!! ERROR: x11-libs/gtk+-2.14.5 failed.
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '208' 'depend'

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090227 failed.
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '176' 'depend'
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              apm2-2.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'

!!! ERROR: sys-mobilephone/fso-raw-1.0.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!   python-emotion-20090614.ebuild, line   5:  Called inherit 'enlightenment_svn-python'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment_svn-python'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 203' for eclass enlightenment_svn-python

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild' P='fsodatad-0.1.0' PF='fsodatad-0.1.0' PN=fsodatadpath
!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'

!!! ERROR: sys-mobilephone/fso-specs-2010.05.11.2 failed.
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!      xglamo-20081205-r2.ebuild, line   5:  Called inherit 'autotools' 'git' 'toolchain-funcs'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!              ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '213' 'depend'
!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.bash, line 209:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/fso-alsa-1.0.0 failed.
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '200' 'depend'

!!! ERROR: sys-mobilephone/fsonetworkd-0.1.0 failed.
!!!   libframeworkd-glib-20090227.ebuild, line   5:  Called inherit 'git' 'autotools'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'

!!! ERROR: sys-mobilephone/fso-abyss-0.9.1 failed.
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                    ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '205' 'depend'

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090614 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '218' 'depend'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!         fsomusicd-0.0.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!              ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '245' 'depend'
!!!           fso-raw-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!              ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'

!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!                      ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libframeworkd-glib-99999999 failed.
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '210' 'depend'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                      ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!       fsonetworkd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-99999999.ebuild' P='libframeworkd-phonegui-99999999' PF='libframeworkd-phonegui-99999999' PN='libframeworkd-phonegui'path
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 306' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild' P='libframeworkd-phonegui-efl-20090506' PF='libframeworkd-phonegui-efl-20090506' PN='libframeworkd-phonegui-efl'path
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!          fso-alsa-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!          isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/fsotdld-0.3.0 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   libframeworkd-glib-20090614.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!                   ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-20090724 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   fso-specs-2010.05.11.2.ebuild, line   6:  Called inherit 'base' 'autotools' 'git'

!!! ERROR: sys-mobilephone/libfsotransport-0.9.6 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                   ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '198' 'depend'
!!!             ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!    ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!               ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!                   ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '266' 'depend'
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                   ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-efl-20090614 failed.
!!!             ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!                           ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
!!!         fso-abyss-0.9.1.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                           ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!             ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '238' 'depend'
!!!                           ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!               isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 223' for eclass git

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.9' PV='0.9.9' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsobasics/libfsobasics-0.9.9.ebuild' P='libfsobasics-0.9.9' PF='libfsobasics-0.9.9' PN=libfsobasicsfailed
!!!                       ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                           ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!           fsotdld-0.3.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                           ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '306' 'depend'
!!!                       ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'

!!! ERROR: sys-mobilephone/libfso-glib-2010.05.11.2 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 251' for eclass cornucopia_git

!!!   libframeworkd-glib-99999999.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!! die invoked from directory /usr/lib64/pkgcore
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui-shr/libphone-ui-shr-99999999.ebuild' P='libphone-ui-shr-99999999' PF='libphone-ui-shr-99999999' PN='libphone-ui-shr'path
!!! Call stack:
.5.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-base/xglamo-20081205-r2
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fsomusicd-0.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-9999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/fsousaged-0.9.4
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-raw-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090724
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsonetworkd-0.1.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-abyss-0.9.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfso-glib-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mioctl-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while pException in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                     ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'
!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 233' for eclass base
!!!                           ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.1' PV='0.1.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsoresource/libfsoresource-0.1.1.ebuild' P='libfsoresource-0.1.1' PF='libfsoresource-0.1.1' PN=libfsoresourcepath
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                                   ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild'
!!!                       ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '286' 'depend'
!!!                     ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'
!!!                                   ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild'
!!!                       ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!                               ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsotransport/libfsotransport-0.9.6.ebuild'
!!!                               ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!!                       isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090614.ebuild'
!!!   libfsotransport-0.9.6.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!   libframeworkd-phonegui-efl-20090614.ebuild, line   8:  Called inherit 'shr'
!!!                ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!                   isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild'
!!!                ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '255' 'depend'
!!!   libframeworkd-phonegui-20090724.ebuild, line   5:  Called inherit 'shr'
!!!             ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                        ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 231' for eclass shr

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-ui/libphone-ui-99999999.ebuild' P='libphone-ui-99999999' PF='libphone-ui-99999999' PN='libphone-ui'path
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                 ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                            ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 246' for eclass shr

!!!                        ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild'
received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-99999999.ebuild' P='libphone-utils-99999999' PF='libphone-utils-99999999' PN='libphone-utils'path
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                        ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!            isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libfso-glib/libfso-glib-2010.05.11.2.ebuild'
!!!   libfso-glib-2010.05.11.2.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!      ebuild-default-functions.lib, line 230:  Called __internal_inherit 'base'
!!!                 ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass base

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2.0.1' PV='2.0.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mdbus2/mdbus2-2.0.1.ebuild' P='mdbus2-2.0.1' PF='mdbus2-2.0.1' PN=mdbus2path
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/eclass/cornucopia_git.eclass'
!!!          cornucopia_git.eclass, line   6:  Called inherit 'base' 'git' 'autotools'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/libphone-utils-20090724 failed.
!!! die invoked from directory /usr/lib64/pkgcore
!!! Call stack:
!!!               ebuild-daemon.bash, line 451:  Called __ebd_exec_main 'daemonize'
!!!               ebuild-daemon.bash, line 209:  Called __ebd_main_loop
!!!               ebuild-daemon.bash, line 437:  Called __ebd_process_metadata '246' 'depend'
!!!               ebuild-daemon.bash, line 370:  Called __execute_phases 'depend'
!!!                       ebuild.lib, line 426:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-20090724.ebuild'
!!!                       ebuild.lib, line 241:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-20090724.ebuild'
!!!           isolated-functions.lib, line 221:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libphone-utils/libphone-utils-20090724.ebuild'
!!!   libphone-utils-20090724.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!     ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 225' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.2' PV='1.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/serial_forward/serial_forward-1.2.ebuild' P='serial_forward-1.2' PF='serial_forward-1.2' PN='serial_forward'failed
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib64/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/repository.py", line 497, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 274, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 365, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 373, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 781, in get_keys
    {"key": receive_key})
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 714, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/usr/lib64/python2.7/site-packages/pkgcore/ebuild/processor.py", line 430, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

rocessing ebuild src: sys-mobilephone/mterm2-1.9.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsoresource-0.1.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-utils-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneui-apps-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mdbus2-2.0.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-20090901
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-utils-20090724
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/serial_forward-1.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/shr-contacts-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/shr-contacts-20090614
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
